docs/translating/polish.md

```diff
@@ -74,10 +78,46 @@ Zulip friendly and usable.
   (za)subskrybować kanał (verb), perfective form depending on the
   context
 
-**topic**: wątek
+**topic**: wątek example:Add new topic - Dodaj nowy wątek
 
 **unsubscribing from a stream**: odsubskrybowanie kanału (noun) and odsubskrybować kanał (verb)
 
 **view**: widok
 
-**starred message**: oznaczona wiadomość
+**starred message**: oznaczona wiadomość (noun) and oznacz wiadomość (verb)
+
+**message**: wiadomość example:You have got new message - Masz nową wiadomość
+
+**public stream**: kanał publiczny
+
+**bot**: bot
+
+**Integration**: integracja
+
+**Notification**: Powiadomienie example: Turn on notification - Włącz powiadomienia
+
+**alert word**: ostrzeżenie
+
+**narrow to**: zwięź do
```

Comment:
I'd use "zawęź", I think you meant "zwęź" which would be used for a finished action, whereas here the action is not yet finished only implied (for non-Polish speakers, we have different verb forms for finished and unfinished actions)